### PR TITLE
Updated Expectations for Vineet Kumar EquityProductModifications

### PR DESCRIPTION
**Pull request by JayasriR**
Original PR: https://github.com/finos/common-domain-model/pull/3174

### CODE REVIEW

File: docs/event-model.md [hunk 1]
```diff
@@ -414,6 +414,7 @@ func Create_TradeState:
   inputs:
     primitiveInstruction PrimitiveInstruction (0..1)
     before TradeState (0..1)
+    eventDate date (0..1)
```
Comment (Oblongs) [inline docs/event-model.md:417]: Would be good to explain this

File: rosetta-source/src/main/rosetta/event-common-func.rosetta [hunk 2]
```diff
@@ -2442,6 +2447,7 @@ func Create_TradeState: <"Creates a single trade state by applying primitive ins
     inputs:
         primitiveInstruction PrimitiveInstruction (0..1) <"The set of primitive instructions to apply to the trade.">
         before TradeState (0..1) <"The original trade on which the primitive instructions are applied">
+        eventDate date (0..1) <"The date associated with the ">
```
Comment (Oblongs) [inline rosetta-source/src/main/rosetta/event-common-func.rosetta:2450]: Comments needs fixing

File: rosetta-source/src/main/rosetta/product-qualification-func.rosetta [hunk 3]
```diff
@@ -966,6 +970,19 @@ func Qualify_EquityOption_ParameterReturnDividend_Basket: <"Qualifies a product
             // qualifies that the underlier is a single name dividend swap
             and Qualify_EquitySwap_ParameterReturnDividend_Basket(underlierEconomicTerms) = True
```
Comment (Oblongs) [inline rosetta-source/src/main/rosetta/product-qualification-func.rosetta:971]: fix the comment

File: rosetta-source/src/main/rosetta/product-template-type.rosetta [hunk 4]
```diff
@@ -335,6 +335,7 @@ type Payout: <"A class to represent the set of future cashflow methodologies in
         if optionPayout exists
         then optionPayout -> priceQuantity exists
                 or optionPayout -> underlier -> contractualProduct -> economicTerms -> payout -> interestRatePayout count = 2
+                or optionPayout -> underlier -> foreignExchange exists
```
Comment (Oblongs) [inline rosetta-source/src/main/rosetta/product-template-type.rosetta:338]: update the comment to match the proposed code change

**Comment by lolabeis** [comment]
Closing, superseded by #3199

**Pull request closed without merging**